pkg/multus/multus_test.go

```diff
@@ -160,718 +181,103 @@ func (f *fakeExec) ExecPlugin(ctx context.Context, pluginPath string, stdinData
 
 	dec := json.NewDecoder(reader)
 	enc := json.NewEncoder(writer)
-	err = dec.Decode(&m)
-	Expect(err).NotTo(HaveOccurred())
-	for k := range m {
-		if k == "prevResult" {
-			delete(m, k)
-		}
-	}
```

Comment:
I don't understand this huge diff until (old) line 817. 

Either I'm missing something or this is the same code as in https://github.com/k8snetworkplumbingwg/multus-cni/pull/785/files#diff-4a220b8816e7d9c591fef947d5847068f9e454f9a2ff482aed709128fabc4c88R184-R192

Could this be just some spaces vs tabs issue ?

Reply:
It is just remove deduped testing.  Check the master branch, you can find several tests are identical, for example "returns the previous result using CmdCheck"

- https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/pkg/multus/multus_test.go#L402
- https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/pkg/multus/multus_test.go#L559
- https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/pkg/multus/multus_test.go#L717
- https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/pkg/multus/multus_test.go#L874
- https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/pkg/multus/multus_test.go#L1031

Reply:
I'm just saying that the code I've linked above (pasting [here](https://github.com/k8snetworkplumbingwg/multus-cni/pull/785/files#diff-4a220b8816e7d9c591fef947d5847068f9e454f9a2ff482aed709128fabc4c88R183-R191) the updated link) seems to be equal before and after this patch - without a line of comment - but is somehow still listed as a difference. I find that odd, and pointed it out.

I'm OK with removing the duplicated tests - but I haven't checked if **all** that is deleted are those tests ( or any other duplicates).

Reply:
I understand the gist of what Tomo's pointing out, which is that there's a number of duplicated tests with the same name. I think it's appropriate to remove them.

But you're right Miguel, it's super hard to read the diff on GH.

Reply:
> I understand the gist of what Tomo's pointing out, which is that there's a number of duplicated tests with the same name. I think it's appropriate to remove them.

I have no issue - at all - about removing the duplicate tests. I want them gone as much as you do. For the sake of clarity: we all agree on that: **no point preserving duplicate tests**. 

I personally would have done it in a separate PR - since it would be a lot easier to review this one -  but I haven't complained about that until now, and I **believe that's besides the point**.

> 
> But you're right Miguel, it's super hard to read the diff on GH.

I'm showing in the links at the bottom of https://github.com/k8snetworkplumbingwg/multus-cni/pull/785#discussion_r800928607 there are **more** changes in the multus tests than just deleting duplicates. All I'm saying is there's something weird there, since the code seems the same to my eyes: it could be a github bug, could be spaces vs tabs, could be anything else. I don't know, but it caught my eye. 

If the maintainers (you) feel / know there's nothing wrong there, by all means, proceed.